benchmark-ph1ex1/3D-model/README.md

```diff
@@ -0,0 +1,38 @@
+This section calculates Phase 1 Exercise 1 of the MHTGR Benchmark.
+Due to a memory problem, the exercise was conducted with reflective boundary conditions instead of periodic ones.
+
+Geometries:
+----------
+* 3Dfullcore-out.msh
+	* Top reflector: 118.9
+	* Bottom reflector: 198.2
+	* Outer reflector: 297.3
+	* h = 20
+
+* 3Dfullcore-in.msh
+	* Top reflector: 118.9
+	* Bottom reflector: 198.2
+	* Outer reflector: 297.3
+	* h = 20
+
+Input files:
+------------
+* 3D-fullcore26G-kout:
+	- Neumann BCs
+	- oecdxsC-12G
+
+* 3D-fullcore26G-kin:
+	- Neumann BCs
+	- oecdxsC-12G
```

Comment:
How are kin and kout different? Is the control rod completely out?

Reply:
> Will you also be generating temperature/flux data in .csv files for plotting? You mentioned plotting in issue #54 and it would be good to check that they're physically sound.

Yes, the ```.csv``` files contain the necessary information for plotting the axially averaged radial power (``` axially_averaged_radial_power_ ```) and the radially averaged axial power (``` radially_averaged_axial_power_ ```).

Reply:
> How are kin and kout different? Is the control rod completely out?

Almost. For Kin, the rod is inserted until half of the height of the bottom reflector. And for Kout, the rod is inserted until the bottom face of the top reflector.